src/main/scala/smack/messaging/Client.scala

```diff
@@ -221,9 +298,17 @@ class Client extends FSM[State, Context] {
   def setupRosterListener(connection: XMPPTCPConnection): Unit =
     Roster.getInstanceFor(connection).addRosterListener(rosterListener)
 
+  def setupDeliveryReceiptManager(connection: XMPPTCPConnection): Unit = {
```

Comment:
It was -- you'll notice there are two open PRs. But the newer PR is built on that branch because there were some necessary changes from master, and so github seems to merge them into one.